src/lib/estimates.js

```diff
@@ -5,18 +5,22 @@
  * @param {Number} latestInterval The interval between the most recent and previous purchases
  * @param {Number} numberOfPurchases Total number of purchases for the item
  */
+
 const calculateEstimate = (lastEstimate, latestInterval, numberOfPurchases) => {
-    if (numberOfPurchases >= 2) {
-        if (isNaN(lastEstimate)) {
-            lastEstimate = 14;
-        }
-        let previousFactor = lastEstimate * numberOfPurchases;
-        let latestFactor = latestInterval * (numberOfPurchases - 1);
-        let totalDivisor = numberOfPurchases * 2 - 1;
-        return Math.round((previousFactor + latestFactor) / totalDivisor);
-    } else {
-        return latestInterval;
-    }
+  if (isNaN(lastEstimate)) {
+    lastEstimate = 14;
+  }
+
+  if (numberOfPurchases >= 2) {
+    let previousFactor = lastEstimate * numberOfPurchases;
+    let latestFactor = latestInterval * (numberOfPurchases - 1);
+    let totalDivisor = numberOfPurchases * 2 - 1;
```

Comment:
Just in case, you don't need `let` here as you're not changing this value at any time, `const` will be just fine.

This doesn't affect the outcome of this operation, I just wanted to point it out!